File: Python-program/Python/marks.py
name = input ("Enter student name: ")

Hindi = int(input("Enter Hindi Marks: "))
English = int(input("Enter English Marks: "))
Math = int(input("Enter Math Marks: "))
Science = int(input("Enter Science Marks: "))
Computer = int(input("Enter Computer Marks: "))

total = Hindi + English + Math + Science + Computer 
percentage = total / 5

print("Result")
print("Name:", name)
print("total Marks: ",total)
print("percentage: ", percentage, "%")

File: c program/c/marks.c
#include<stdio.h>
int main()
{
    int hindi;
    int urdu;
    int english;
    int math;
    int science;
    int total;
    float percentage;
    printf("please enter marks in hindi: ");
    scanf("%d",&hindi);

    printf("please enter marks in urdu: ");
    scanf("%d",&urdu);

    printf("please enter marks in english: ");
    scanf("%d",&english);

    printf("please enter marks in math: ");
    scanf("%d",&math);

    printf("please enter marks in science: ");
    scanf("%d",&science);

    total=hindi+urdu+english+math+science;
    percentage=total/5;

    

    if(percentage>=60)
    {
        printf("I division");

    }
    else if(percentage>=45 && percentage<60)
    {
        printf("II division ");
    
    }
    else if (percentage>=33 && percentage<45)
    {
        printf("III division ");
    }
    else 
    {
        printf("failed");
    }

    return 0;


}